src/detect-lua.h

```diff
@@ -32,6 +32,7 @@ typedef struct DetectLuaThreadData {
     lua_State *luastate;
     uint32_t flags;
     int alproto;
+    bool error_logged;
```

Comment:
guess this could also be done as a flag in the flags field? Leaves struct slightly more compact